docs/proposals/kubeedgeadm-scope.md

```diff
@@ -29,13 +29,13 @@ Hence proposing the following commands for KubeEdge installation process.
 
 For cloud, commands shall be:
```

Comment:
can we rename the document to keadm.md ?

Reply:
Thanks. Done